src/data/data_services.py

```diff
@@ -96,12 +119,41 @@ def fetch_ticker_financials_by_year(self, year: int, ticker: str = None) -> None
         Returns:
             None
         """
-        # TODO: skip in a better way. For now skip these ETFs manually
         if ticker in ['spy', 'qqq']:
             return
 
-        if self.da.is_ticker_stored(ticker, year):
+        if self.data_access.is_ticker_stored(ticker, year):
             logging.info(
                 f'data_assets is already cached data_assets for {ticker} {year}')
             return
-        self.sg.fetch_ticker_financials_by_year(year, ticker)
+        self.sec_gov.fetch_ticker_financials_by_year(year, ticker)
+
+
+ds = DataServices()
```

Comment:
```suggestion
data_service = DataServices()
```
Will be clearer and easier to maintain later on